docs/Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md

```diff
@@ -974,6 +974,26 @@ privacy-enabled=false
 Set to enable [private transactions](../../Concepts/Privacy/Privacy-Overview.md). 
 The default is false.  
 
+### privacy-marker-transaction-signing-key-file
+
+```bash tab="Syntax"
+--privacy-marker-transaction-signing-key-file=<FILE>
+```
+
+```bash tab="Command Line"
+--privacy-marker-transaction-signing-key-file=/home/me/me_node/myPrivateKey
+```
+
+```bash tab="Environment Variable"
+PANTHEON_PRIVACY_MARKER_TRANSACTION_SIGNING_KEY_FILE=/home/me/me_node/myPrivateKey
+```
+
+```bash tab="Configuration File"
+privacy-marker-transaction-signing-key-file="/home/me/me_node/myPrivateKey"
+```
+
+`<FILE>` is the name of the private key file used to sign Privacy Marker Transactions. If unset, each transaction is signed with a different randomly generated key.
```

Comment:
```suggestion
`<FILE>` is the name of the private key file used to sign Privacy Marker Transactions. If this command line isn't specified, each transaction is signed with a different randomly generated key.
```

I'm not sure 'If unset' makes it clear enough that you leave out the whole CLI option rather than just the <FILE>

Reply:
I think this section needs a link to the How To topic on signing Privacy Marker Transactions.  If a user starts at the tutorial, then comes here, there's no link to anything further.